pulpcore/cli/config.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import click
+import toml
+
+DEFAULT_PATH = os.path.expanduser("~/.config/pulp/settings.toml")
```

Comment:
Can we try `$XDG_CONFIG_HOME` first?

https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html